CDTDatastore/CDTReplicator/CDTReplicator.m

```diff
@@ -109,26 +110,51 @@ + (NSString *)stringForReplicatorState:(CDTReplicatorState)state
     }
 }
 
++ (CDTReplicator *)replicatorWithConfiguration:(CDTAbstractReplication *)configuration
```

Comment:
It probably will, since it will be the first place they look, so it should be removed.